cpp/include/cudf/strings/json.hpp

```diff
@@ -17,6 +17,8 @@
 
 #include <cudf/strings/strings_column_view.hpp>
 
+#include <thrust/optional.h>
```

Comment:
Should we convert this to std::optional now that we support c++17?

Reply:
Agree 😄